manim_slides/slide.py

```diff
@@ -53,6 +53,7 @@ def __init__(
         self.__current_animation = 0
         self.__loop_start_animation: Optional[int] = None
         self.__pause_start_animation = 0
+        self.__background_color = config["background_color"].hex
```

Comment:
This does not work because the `background_color` attribute is not placed in `config` with ManimGL. So you need to a separate implementation, as with `__resolution`:

https://github.com/jeertmans/manim-slides/blob/2a327c470bcea5821f4ff21cf1085a81587b745c/manim_slides/slide.py#L57-L63

Reply:
Oh, I Didn't know, understood!